### ISSUE TITLE
Exhibits: Bookmark ribbons for in-book images in galleries

### ISSUE THREAD

**Issue by zploskey** [zploskey/ExperimentalBeijing#32]
Should be a little overlay image shown over one of the upper corners. Maybe png or svg. Check what's available online or create one.

**Comment by zploskey** [comment]
Actually we can totally use this glyph:
https://fontawesome.com/icons/bookmark?style=solid

**Comment by zploskey** [comment]
@swelland, The mock-up has the following being displayed over the thumbnail on hover when it has a location in the book.

```
Title (Date)
Page 01
```

Are you happy with that? Should we be showing Chapter, Plate or Figure info here as well?

On a related noted, the display of the Book Reference looks a bit odd on item pages. For example,
http://experimentalbeijing.com/items/show/679 displays this:

Book Reference:	 Figure #intro.01, Chapter #introduction

What would be your preferred way to display that line (including a page number)?

**Comment by swelland** [comment]
Hi Zach,

I’m just prepping for class but will give some thought to these questions this afternoon/evening and get back to you ASAP. Now that my spring quarter course is up and running, I’ve been starting to work on the content side. I got your earlier github note about the bookmark icons. In case they are helpful, Siyi also sent along the attached, with colors she though could work with the final print cover design. 

More soon,
Sasha

**Comment by zploskey** [comment]
Can you send those in a separate email? Github strips out attachment. Thanks!

**Comment by swelland** [comment]
Zach,

I see that you were already able to do some work on this last night. 

I think that for the hover-over, it’s best to just include what Siyi included in the design:

Title (Date)
Page 01

I think that including the other book reference material makes it too cluttered.

I will include page numbers for some items that are referenced in the text but are not printed in the book as a plate or figure. The bookmark ribbon is to indicate that the image appears in the book. Therefore, the bookmark ribbons should appear IF either Plate or Figure metadata fields have a value in them; the Page field should not be a determining factor.

I had noticed the odd Book Reference display; thanks for addressing that. I give two examples below of how I think it should read:

Book Reference: Chapter 1, Figure 1.04, Page 44
Book Reference: Introduction, Plate 2, Page 15

I will fill out the Chapter metadata field as follows: Prologue, Introduction, Chapter 1, Chapter 2…Epilogue.
So for that field, you do not need to include “Chapter” within the display code. 

For the Figure, Plate and Page metadata fields, I will just enter a number.

I was working on the About exhibit pages yesterday and for example added the book cover to the “About the Book” page, with not caption. Between then and now, a clickable caption (to the Item page) has begun appearing below the image. I can’t figure out why or how to remove this. I’ll add this issue to GitHub.

The book is already out, so I’ve added this under construction caveat <http://experimentalbeijing.com/exhibits/show/about>. I will make a push in the next weeks on the content side. Can we set a meeting sometime in April or early May to review things and work out final design issues?

Thanks,
Sasha
